client/src/Components/CommonComponents/OfferCard/style.css

```
@@ -0,0 +1,161 @@
+.offer-card {
+  margin: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  color: rgba(255, 255, 255, 0);
+  cursor: pointer;
+}
+  
+.offer-card:hover {
+  box-shadow:  0 0 30px -15px rgba(0, 0, 0, 0.3)
+}
+
+.offer-card--header {
```

header isn't a modifier
```suggestion
.offer-card__header {
```